lib/NetworkManager.js

```diff
@@ -151,9 +151,10 @@ class NetworkManager extends EventEmitter {
 
     if (event.redirectUrl) {
       const request = this._interceptionIdToRequest.get(event.interceptionId);
-      console.assert(request, 'INTERNAL ERROR: failed to find request for interception redirect.');
-      this._handleRequestRedirect(request, event.responseStatusCode, event.responseHeaders);
-      this._handleRequestStart(request._requestId, event.interceptionId, event.redirectUrl, event.resourceType, event.request, event.frameId);
+      if (request) {
```

Comment:
fast return?

Reply:
Is there a todo here after servicification happens? I'm not too familiar with this part of the code base.

Reply:
> fast return?

I hope you don't mind if I keep this as-is.

> Is there a todo here after servicification happens? I'm not too familiar with this part of the code base.

There's an implicit "todo": the `interceptionId` will go away, so we'll basically have to rewrite the NetworkManager.